lib/ckb_widget_localization_delegate.dart

```diff
@@ -16,7 +16,7 @@ class _CkbMaterialLocalizationsDelegate
   @override
   Future<WidgetsLocalizations> load(Locale locale) async {
     const String localeName = "ckb";
-    await intl.initializeDateFormatting(localeName, null);
+    await intl.initializeDateFormatting(localeName);
```

Comment:
Also, this line gave me errors and i just removed it for my self. Didn't get into it too much, but it works for me without it.